src/displayapp/screens/Tile.cpp

```diff
@@ -22,10 +20,12 @@ Tile::Tile(DisplayApp* app, std::array<Applications, 6>& applications) : Screen(
       appIndex++;
     }
   }
-  modal.reset(new Modal(app));
+  
+  // ????
```

Comment:
????

Reply:
this call to the modal must have been lost code, the ????? was to check later ....